.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -234,9 +234,14 @@ run_tests() {
 
   droute_send $release_name $project
 
-  exit ${RESULT}
+  echo ${RESULT}
+
+  if [ ${RESULT} -ne 0 ]; then
+    exit ${RESULT}
```

Comment:
In this case, if the showcase project fails, the RBAC test will not be executed as it will simply exit.

Reply:
@subhashkhileri do you have a suggestion ?